### PR TITLE
tvOS Example

### PR DESCRIPTION
**Pull request by kirakatzen**
Example target for tvOS app. It reuses iOS module.

### CODE REVIEW

File: TV/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
+    }
+
+    func applicationWillResignActive(_ application: UIApplication) {
```
Comment (khramtsoff) [inline TV/AppDelegate.swift]: Please remove empty methods

**Review by khramtsoff — CHANGES_REQUESTED** [review]

File: TV/AppDelegate.swift (hunk 1 above)
Comment (kirakatzen) [reply TV/AppDelegate.swift]: Fixed

**Review by khramtsoff — DISMISSED** [review]

File: TV/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
```
Comment (aeternas) [inline TV/AppDelegate.swift]: Should we remove this line for the sake of perfection?

Comment (kirakatzen) [reply TV/AppDelegate.swift]: Fixed!

**Review by aeternas — APPROVED** [review]

**Review by imJustik — APPROVED** [review]

**Pull request merged**